### PR TITLE
Fix for longwave and integer division issues

### PR DESCRIPTION
**Pull request by jordi-prats**
I think the correct value for the cloud coefficient should be 1.24 instead of 0.642 in the long wave calculations using Brutsaert and Yajima formulae.
Also, I found a pair of integer division issues in the code.

See here for more details:
https://groups.google.com/d/msg/aquaticmodelling/AMGGI73kLV8/-IxpTpVYCgAJ

### CODE REVIEW

File: src/glm.h [hunk 1]
```diff
@@ -35,7 +35,7 @@
 #endif
 #define USE_FILLVALUE 1
 
-#define GLM_VERSION  "2.2.0rc4"
+#define GLM_VERSION  "2.2.0"
```
Comment (lawinslow) [inline src/glm.h]: Could you increment this to 2.2.1 instead of dropping it to zero?

**Review by lawinslow — COMMENTED** [review]
Overall, you've made a lot of changes on the build side here that go beyond the coefficient and integer division issues. I think it is important to incorporate the coefficient fixes, but am uncertain if the build changes are universally important (I don't build in Windows or Mac). 

Overall, I say we

1. Change version change to 2.2.1 to avoid ambiguity. 

2. Justify or remove the build-side changes. 

Would that be possible?

**Comment by jordi-prats** [comment]
Actually, I didn't do the build changes. They were made by Casper Boon and Matt Hipsey and they were already there the last time I pulled from AED/GLM (I might have pulled somewhat later than you, so I had some more commits). I cannot tell you about the importance of the build changes either.
I agree about changing the version to 2.2.1.

**Comment by lawinslow** [comment]
Ah, ok. Should have looked at where those changes came from. Let's just merge them all in. want to push that version increment and I'll merge?

**Comment by lawinslow** [comment]
Awesome. Thanks so much for the PR on this. It's great to have people diving into the code.

**Pull request merged**